slice/IceGrid/Session.ice

```diff
@@ -17,53 +17,49 @@
 
 module IceGrid
 {
-    interface Registry; // So that doc-comments can link to `IceGrid::Registry`.
+    interface Registry; // For doc-comments.
 
-    /// A session object is used by IceGrid clients to allocate and release objects. Client sessions are created either
-    /// via the {@link Registry} object or via the registry client <code>SessionManager</code> object.
-    /// @see Registry
+    /// Represents a session object used by IceGrid clients to allocate and release objects. Client sessions are created
+    /// either via the {@link Registry} object or via the registry client `SessionManager` object.
     interface Session extends Glacier2::Session
     {
-        /// Keep the session alive.
-        /// As of Ice 3.8, there is no need to call this operation, and its implementation does nothing.
+        /// Keeps the session alive.
+        /// @deprecated As of Ice 3.8, there is no need to call this operation, and its implementation does nothing.
```

Comment:
Should add a comment to the metadata.

Reply:
Fixed. All removed @deprecated doc-comments.